Formula/dosfstools.rb

```diff
@@ -27,18 +19,26 @@ class Dosfstools < Formula
 
   depends_on "autoconf" => :build
   depends_on "automake" => :build
+  depends_on "gettext" => :build
   depends_on "pkg-config" => :build
 
+  # remove in next release
+  patch do
+    url "https://github.com/chenrui333/dosfstools/commit/8a917ed.patch?full_index=1"
```

Comment:
```suggestion
  # remove in next release
  # https://github.com/dosfstools/dosfstools/pull/158
  patch do
    url "https://github.com/dosfstools/dosfstools/commit/8a917ed2afb2dd2a165a93812b6f52b9060eec5f.patch?full_index=1"
```
Let's use the commit in your PR.